packages/terminal/src/widget.ts

```diff
@@ -141,13 +126,12 @@ export class Terminal extends Widget {
   /**
    * Refresh the terminal session.
    */
-  refresh(): Promise<void> {
+  async refresh(): Promise<void> {
     if (!this._session) {
       return Promise.reject(void 0);
```

Comment:
ooh, i realize this isn't actually caught out in the await...

Reply:
I'm not sure I follow...